src/components/Story/StoryPresentation.js

```diff
@@ -108,13 +112,13 @@ function StoryPresentation({ entry }) {
                   <div className="w-full md:text-lg text-fv-charcoal-light">
                     <WysiwygBlock jsonString={entry?.introTranslation} />
                   </div>
-                  {entry?.audio?.length > 0 && (
+                  {entry?.relatedAudio?.length > 0 && (
                     <div className="space-y-5">
-                      {entry?.audio?.map((audioId) => (
+                      {entry?.relatedAudio?.map((audio) => (
                         <AudioNative
-                          key={`${audioId}}`}
+                          key={audio.id}
                           styling="lg:w-96 print:hidden"
-                          audioId={audioId}
+                          audioId={audio.id}
                         />
```

Comment:
AudioNative is still using the old nuxeo techniques. @gmcauliffe Is there already a replacement for this or should I update it in a new PR?

Reply:
There isn't a replacement for that yet, but check in with @sahil97 because I think he may be creating one as part of the media tickets he is currently working on.